src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -0,0 +1,133 @@
+        {
+            Guard.NotNull(logger, nameof(logger));
+            Guard.NotNull(request, nameof(request));
+            Guard.NotNull(response, nameof(response));
+
+            context = context ?? new Dictionary<string, object>();
+
+            var statusCode = response.StatusCode;
+            var resourcePath = request.Path;
+            var host = $"{request.Scheme}://{request.Host}";
+
+            logger.LogInformation(
+                $"{MessagePrefixes.RequestViaHttp} {{{ContextProperties.RequestTracking.RequestMethod}}} {{{ContextProperties.RequestTracking.RequestHost}}}/{{{ContextProperties.RequestTracking.RequestUri}}} completed with {{{ContextProperties.RequestTracking.ResponseStatusCode}}} in {{{ContextProperties.RequestTracking.RequestDuration}}} at {{{ContextProperties.RequestTracking.RequestTime}}} (Context: {{@{ContextProperties.EventTracking.EventContext}}})",
+                request.Method,
+                host, resourcePath, statusCode, duration, DateTimeOffset.UtcNow, context);
+        }
+
+        /// <summary>
+        ///     Logs an HTTP dependency
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="request">Request that started the HTTP communication</param>
+        /// <param name="statusCode">Status code that was returned by the service for this HTTP communication</param>
+        /// <param name="startTime">Point in time when the interaction with the HTTP dependency was started</param>
+        /// <param name="duration">Duration of the operation</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogHttpDependency(this ILogger logger, HttpRequestMessage request, HttpStatusCode statusCode, DateTimeOffset startTime, TimeSpan duration, Dictionary<string, object> context = null)
+        {
+            Guard.NotNull(logger, nameof(logger));
+            Guard.NotNull(request, nameof(request));
+
+            context = context ?? new Dictionary<string, object>();
+
+            var requestUri = request.RequestUri;
+            var targetName = requestUri.Host;
+            var requestMethod = request.Method;
+            var dependencyName = $"{requestMethod} {requestUri.AbsolutePath}";
+            var isSuccessful = (int)statusCode >= 200 && (int)statusCode < 300;
+
+            logger.LogInformation(
+                $"{MessagePrefixes.DependencyViaHttp} {{{ContextProperties.DependencyTracking.TargetName}}} for {{{ContextProperties.DependencyTracking.DependencyName}}} completed with {{{ContextProperties.DependencyTracking.ResultCode}}} in {{{ContextProperties.DependencyTracking.Duration}}} at {{{ContextProperties.DependencyTracking.StartTime}}} (Successful: {{{ContextProperties.DependencyTracking.IsSuccessful}}} - Context: {{@{ContextProperties.EventTracking.EventContext}}})",
```

Comment:
The message prefix also in the formatting template instead of concatenating?

Reply:
This is intentional as we want to use the value here while with the other approach we want to use structured logging.

This results in : `Dependency {DependencyName}` while with your suggestion it would be ``{DependencyType} {DependencyName}`.

if it's ok for you I'd like to keep it as is and revise when I add the converters, is that ok or do you want to change this perse?

Reply:
Aha is good